### PR TITLE
[EWPP-1194] EWPP-1195: Adds Contact paragraph.

### PR DESCRIPTION
**Pull request by 22Alexandra**

### CODE REVIEW

File: modules/oe_paragraphs_contact/config/install/field.field.paragraph.oe_contact.field_oe_title.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    - field.storage.paragraph.field_oe_title
+    - paragraphs.paragraphs_type.oe_contact
+id: paragraph.oe_paragraph_contact.field_oe_title
+field_name: field_oe_title
+entity_type: paragraph
+bundle: oe_contact
+label: Heading
+description: ''
+required: false
+translatable: true
```
Comment (nagyad) [inline modules/oe_paragraphs_contact/config/install/field.field.paragraph.oe_contact.field_oe_title.yml:14]: Translatable storage was in the ticket only.

File: tests/Behat/DrupalContext.php [hunk 2]
```diff
@@ -122,4 +126,21 @@ public function assertAvPortalVideoIframe(string $title): void {
     }
   }
 
+  /**
+   * Presses button with specified id|name|title|alt|value on a given region.
```
Comment (nagyad) [inline tests/Behat/DrupalContext.php:130]: Presses a button with the specified....

File: tests/features/contact.feature [hunk 3]
```diff
@@ -0,0 +1,33 @@
+@api
+Feature: Contact paragraph.
+  As a content editor
+  I need to be able to use Contact paragraphs
+  so that I can reference General and Press Contact entities.
+
+  Scenario: Contact creation.
```
Comment (nagyad) [inline tests/features/contact.feature:7]: Editors can create contacts though paragraphs.

Comment (nagyad) [reply tests/features/contact.feature:7]: Test is missing that both press and general contact types are available.

**Review by nagyad — CHANGES_REQUESTED** [review]

File: modules/oe_paragraphs_contact/config/install/field.field.paragraph.oe_contact.field_oe_title.yml (hunk 1 above)
Comment (22Alexandra) [reply modules/oe_paragraphs_contact/config/install/field.field.paragraph.oe_contact.field_oe_title.yml:14]: Indeed but following the requirements [page](https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?pageId=616366260), the field should be translatable and as per Contact field notes, also that one should be translatable. Checking the other fields it seems we're setting them translatable when needed including the field config, not only the storage.

File: tests/Behat/DrupalContext.php (hunk 2 above)
Comment (22Alexandra) [reply tests/Behat/DrupalContext.php:130]: The short description should fit in the 80 chars limit and it wouldn't by changing it as requested.

File: tests/features/contact.feature (hunk 3 above)
Comment (22Alexandra) [reply tests/features/contact.feature]: The select doesn't have a label so I used the select name.

File: tests/features/contact.feature [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    # Check the required field validation.
+    When I press "Save"
+    Then I should see the following error messages:
+      | error messages              |
+      | Contacts field is required. |
+    When I fill in "Heading" with "Contact paragraph demo"
+    And I press "Add new Contact"
+    And I fill in "Name" with "First Contact"
+    And I press "Create Contact"
+    And I select "Press" from "field_oe_demo_body[0][subform][field_oe_contacts][actions][bundle]"
```
Comment (nagyad) [inline tests/features/contact.feature:26]: We should make this readable by making an element out of it and use that here.

**Review by nagyad — CHANGES_REQUESTED** [review]

**Review by nagyad — DISMISSED** [review]

**Review by upchuk — CHANGES_REQUESTED** [review]

**Review by upchuk — CHANGES_REQUESTED** [review]
Looks fine for me, keeping open as it needs to be merged together with the epic.

**Review by yenyasinn — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

**Review by upchuk — APPROVED** [review]

**Pull request merged**